index.html

```diff
@@ -6,16 +6,29 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">            
     <link rel="stylesheet" href="./css/homepage.css">
+    <link rel="shortcut icon" href="./images/d.png" type="image/x-icon">
     <title>Homepage- The 2021 Design & Content Conference</title>
 </head>
 <body>
   <header>
+    <div class="user-bar">
+      <img class="user-bar-smedia" src="./images/facebook.png" alt="facebook logo">
+      <img class="user-bar-smedia" src="./images/twitter.png" alt="twitter logo">
+      <a class="user-bar-link" href="#">English</a>
+      <a class="user-bar-link" href="#">My page</a>
+      <a class="user-bar-link" href="#">Sign in</a>
+    </div>
     <nav>
+      <a class="logo" href="#main">Design & Content</a>
       <img class="hamb-icon" src="./images/menu.png" alt="menu icon">
       <ul class="menu-links">
         <li class="menu-list close"><img class="hamb-close-btn" src="./images/close.png" alt="close button"></li>
-        <li class="menu-list"><a class="menu-link" href="./index.html">Home Page</a></li>
-        <li class="menu-list"><a class="menu-link" href="./html/about.html">About</a</li>
+        <li class="menu-list homepage"><a class="menu-link" href="./index.html">Home Page</a></li>
```

Comment:
- [ ] In the mobile version there is no button to go back to the first page please add it in your hamburger

Reply:
done it tooo